src/main/target/link/stm32_flash_f303_128k.ld

```diff
@@ -14,7 +14,8 @@ MEMORY
 {
   FLASH  (rx)     : ORIGIN = 0x08000000, LENGTH = 126K /* last 2kb used for config storage */
   RAM    (xrw)    : ORIGIN = 0x20000000, LENGTH = 40K
+  CCM    (xrw)    : ORIGIN = 0x10000000, LENGTH = 8K
   MEMORY_B1 (rx)  : ORIGIN = 0x60000000, LENGTH = 0K
 }
 
-INCLUDE "stm32_flash.ld"
+INCLUDE "stm32_flash_ccm.ld"
```

Comment:
Do we need to add a new linker script? Can't we just check if CCM is defined in the existing script?